lib/jekyll-admin/server/page.rb

```diff
@@ -20,7 +20,9 @@ class Server < Sinatra::Base
         end
 
         write_file(write_path, page_body)
-        updated_page = pages.find { |p| p.path == write_path }
+        updated_page = Jekyll::Page.new(
+          site, site.source, File.dirname(write_path), File.basename(write_path)
```

Comment:
Is the path sanitized prior to being read? (to prevent directory traversal from the user-supplied `write_path`?